spec/models/user_spec.rb

```diff
@@ -33,14 +32,19 @@
     context '.after_save' do
       let!(:user) { create(:user) }
 
-      before do
-        user.accounts.create({ name: 'Emprego CLT', active: false }, CltWorkerAccount)
-        user.accounts = [user.accounts.not.active.first]
-        user.save
+      context 'with accounts' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.